src/network-manager/azext_network_manager/_params.py

```diff
@@ -354,6 +356,10 @@ def load_arguments(self, _):
         c.argument('network_manager_name', type=str, help='The name of the network manager.', id_part='name')
         c.argument('configuration_name', type=str, help='The name of the network manager security Configuration.',
                    id_part='child_name_1')
+        c.argument('force', options_list=['--force-delete', '-f'], arg_type=get_three_state_flag(),
+                   help='Deletes the resource even if it is part of a deployed configuration.')
+        c.argument('recursive', options_list=['--recursive-delete', '-f'], arg_type=get_three_state_flag(),
```

Comment:
Why not use the default option `--recursive`?
BTW, '-f' is used in 'force' argument. You should use another one or remove it.